api/v1alpha1/registeredservice_types.go

```diff
@@ -25,26 +25,67 @@ type RegisteredServiceConstraints struct {
     Environments []string `json:"environments,omitempty"`
 }
 
+type HealthCheckContainer struct {
```

Comment:
nit: Add documentation for this struct and the others you added.  It's useful in some scenarios (example: it's what's used for `kubectl explain`)

Reply:
Got it